app/src/androidTest/java/com/example/henzoshimada/feeltrip/FeelTripTests.java

```diff
@@ -174,11 +213,11 @@ public void testGetDate() {
 
 /*
     public void testSetImageFile() {
-        //TODO: Figure out what data type an image file is stored in
```

Comment:
We know what type it is, it's a String. And String types are trivial to make getters and setters for. Not much need to test this, although we could always in the future.